magda-web-client/src/config.ts

```diff
@@ -11,6 +11,28 @@ declare global {
     }
 }
 
+// https://www.w3.org/TR/NOTE-datetime
```

Comment:
Can we make this overridable by server config like a few other of the things below?

Sorry to keep dragging out the PR! 😬